docs/why-dusty.md

```diff
@@ -0,0 +1,66 @@
+# Why Dusty?
+
+We built Dusty to be a great way to run a development environment, and we've tried to learn from
+the existing solutions out there and make improvements where we could. Here, we'll briefly go over
+a few reasons why we think you'll love Dusty more than the alternatives.
+
+## vs. Vagrant
+
+[Vagrant](https://www.vagrantup.com) is a very powerful tool that makes it easy to provision a
+virtual machine. Vagrant is often used in conjunction with configuration management software
+like Chef or Puppet in order to coordinate state changes across many users.
+
+Compared to Vagrant, Dusty has the following advantages:
+
+* **Isolation**: With Docker containers, each service you run is fully isolated. If you have three versions of
+Postgres running across production, Dusty can easily emulate that for you on your local environment. This would
+be very difficult with most Vagrant setups.
+* **Ease of Updating**: By default, Dusty makes sure it has the latest version of its specs whenever you
+run `dusty up`. If someone changes a service in your stack and pushes a change to the specs, Dusty will set that up
+for you automatically.
+* **Efficiency**: Dusty's mix and match capability allows you to run only the apps you need at any given
+time. Switching apps is very easy and supported out of the box. To do this in Vagrant would require heavy
+scripting through a configuration management tool.
+
+## vs. Docker Compose
+
+We love [Docker Compose](https://docs.docker.com/compose/), and Dusty actually uses Compose to orchestrate
+container lifecycles. By itself, however, Compose has some shortcomings that we've tried to address with Dusty.
+
+* **OS X Support**: Using Docker on OS X introduces a virtual machine between your host OS and your containers,
+and navigating that extra layer is often painful. It takes a lot of effort to see a container's web service in a
+browser running on your Mac, or to get files between your Mac and the container. Dusty solves these problems (see
+following section on OS X Support).
+* **Simplified Specs**: Compose's specs model is fairly static. You write a large Composefile with all the containers for a
+specific bundle of apps. It's difficult to share service configuration across these large files. Dusty simplifies the
+specs model: each container is defined in exactly one place, then specs are stitched together at runtime based on the
+dependencies for the bundles you want to run.
```

Comment:
How about `Dusty simplifies the
specs model: each container is defined in exactly one place, then the specs you need are stitched together at runtime.`